File: hw/chavez/u3/hw24CompositePaterns/JavaScript/controller/EmployeeController.js
import Manager from "../model/Manager.js";
import President from "../model/President.js";
import Teller from "../model/Teller.js";
import Clerk from "../model/Clerk.js";

export default class EmployeeController {

    createOrganization() {

        const able = new Manager("Able");

        able.add(new Teller("Lonny"));
        able.add(new Clerk("Cal"));

        const becky = new Manager("Becky");

        becky.add(new Teller("Juanita"));
        becky.add(new Teller("Tina"));

        const pete = new President("Pete");

        pete.add(able);
        pete.add(becky);

        return pete;
    }
}

File: hw/chavez/u3/hw25CompositePaterns/Python/controller/employeeController.py
from model.manager import Manager
from model.president import President
from model.teller import Teller
from model.clerk import Clerk


class EmployeeController:

    def create_organization(self):

        able = Manager("Able")

        able.add(Teller("Lonny"))
        able.add(Clerk("Cal"))

        becky = Manager("Becky")

        becky.add(Teller("Juanita"))
        becky.add(Teller("Tina"))

        pete = President("Pete")

        pete.add(able)
        pete.add(becky)

        return pete